### PR TITLE
SDPA-141 : Add an option to filter search by 'Editor's choice' tag

### PR DESCRIPTION
**Pull request by naomiaro**

### CODE REVIEW

File: scripts/superdesk-search/search.js [hunk 1]
```diff
@@ -2659,7 +2659,14 @@
                          */
                         function getQuery() {
                             var metas = [];
+
                             angular.forEach(scope.meta, function(val, key) {
+                                //checkbox boolean values.
+                                if (typeof(val) === 'boolean') {
+                                    val = Number(val);
```
Comment (ahilles107) [inline scripts/superdesk-search/search.js]: it can't be ```val = Number(val).toString()```?

Comment (naomiaro) [inline scripts/superdesk-search/search.js]: it could... I was debating to change

Comment (petrjasek) [inline scripts/superdesk-search/search.js]: it can, but what id does?

Comment (petrjasek) [inline scripts/superdesk-search/search.js]: imo it should be more like ```val = convertBooleanToBinary(val);```

Comment (ahilles107) [inline scripts/superdesk-search/search.js]: convertBooleanToBinary sounds good ;) It's small PR so comments are also on *microlevel* ;)

Comment (naomiaro) [inline scripts/superdesk-search/search.js]: Yeah, in PA images API the Boolean types needed are 0/1.

I don't mind micro level comments :smile:

**Comment by ahilles107** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

**Comment by ioanpocol** [comment]
Please rebase and create a peer PR on superdesk/pa-poc like this one: 
https://github.com/superdesk/superdesk/pull/1824

**Comment by mugurrus** [comment]
:+1:

**Pull request merged**